server/protocols/jmap/src/test/java/org/apache/james/jmap/json/ParsingWritingObjectsTest.java

```diff
@@ -73,16 +73,6 @@ public void writingJsonShouldWorkOnSubMessage() throws Exception {
     }
 
     @Test
-    public void parsingJsonShouldWorkOnMessage() throws Exception {
```

Comment:
As I added the comment on its commit. The Message model does not using for binding any input data, we had CreationMessage and UpdateMessagePatch for that. So when we simplify the Message model, its deserialization is wrong. That's why I would like to remove that testing